tests/test_response.py

```diff
@@ -168,20 +170,39 @@ def test_stream_response_includes_chunked_header():
 
 def test_stream_response_writes_correct_content_to_transport(streaming_app):
     response = StreamingHTTPResponse(sample_streaming_fn)
-    response.transport = MagicMock(asyncio.Transport)
+    response.protocol = MagicMock(HttpProtocol)
+    response.protocol.transport = MagicMock(asyncio.Transport)
+
+    async def mock_drain():
+        pass
+
+    def mock_push_data(data):
+        response.protocol.transport.write(data)
+
+    response.protocol.push_data = mock_push_data
+    response.protocol.drain = mock_drain
 
     @streaming_app.listener('after_server_start')
     async def run_stream(app, loop):
         await response.stream()
-        assert response.transport.write.call_args_list[1][0][0] == (
+        # assert response.protocol.push_data.call_args_list[1][0][0] == (
```

Comment:
these comments seem accidental -- did you mean to leave them here? They seem to be the exact same as the tests themselves

Reply:
TBH, I don't remember.
They look accidental. I will remove them.